src/analyserbeats.cpp

```diff
@@ -33,14 +33,16 @@ AnalyserBeats::AnalyserBeats(ConfigObject<ConfigValue>* pConfig)
 AnalyserBeats::~AnalyserBeats() {
 }
 
-bool AnalyserBeats::initialise(TrackPointer tio, int sampleRate, int totalSamples) {
+bool AnalyserBeats::initialise(TrackPointer tio, int sampleRate,
+                               int totalSamples) {
     if (totalSamples == 0) {
         return false;
     }
 
     bool bPreferencesBeatDetectionEnabled = static_cast<bool>(
-        m_pConfig->getValueString(
-            ConfigKey(BPM_CONFIG_KEY, BPM_DETECTION_ENABLED)).toInt());
+            m_pConfig->getValueString(
+                             ConfigKey(BPM_CONFIG_KEY, BPM_DETECTION_ENABLED))
+                    .toInt());
```

Comment:
We can tweak the costs of each kind of break to try and avoid this. By default clang-format prefers breaking fewer parentheses so it kept ConfigKey together and dropped toInt to the next line instead of breaking ConfigKey and putting toInt at the end of the 2nd line. 

Basically, if it's automatic I don't really mind it.